queries.sql

```diff
@@ -0,0 +1,16 @@
+/*Quels festivals se déroulent dans une ville spécifique ?*/
+
+
+/*Quel est le visiteur ayant écrit le plus d'avis sur les bières ?*/
+
+
+/*Quels types de bières ont une moyenne d'alcool supérieure à 5% ?*/
+
+
+/*Quels sont les bars ayant une note moyenne supérieure à 4 ?*/
+
+
+/*Quels sont les bars ayant une note moyenne supérieure à 4 ?*/
+
+
+/*Quel est la bière avec le meilleur avis ?*/
```

Comment:
Could you translate the questions to english please ? 
And the 4th question is duplicated!